### ISSUE TITLE
Should report a missing file from tsconfig.json

### ISSUE THREAD

**Issue by tkrotoff** [ivogabe/gulp-typescript#137]
Like tsc would do:
```Shell
$ tsc
error TS6053: File 'FooBar.ts' not found.
```

tsconfig.json:
```JSON
{
  "compilerOptions": {
    "target": "ES5",
    "noImplicitAny": true,
    "removeComments": true,
    "declaration": true,
    "sourceMap": true,
    "out": ".tmp/app.js"
  },

  "files": [
    "FooBar.ts"
  ]
}
```

**Comment by ivogabe** [comment]
Thanks for the suggestion, I'll try to take a look at this soon.
